ansible/eos_install_config

```diff
@@ -42,26 +45,39 @@ requirements:
 options:
     hostname:
         description: IP or FQDN of the device you want to connect to
-        required: true
+        required: True
     username:
         description: Username
-        required: true
+        required: True
     password:
         description: Password
-        required: true
+        required: True
     use_ssl:
```

Comment:
We should actually remove that option and make everybody use SSL always. Why? Napalm doesn´t support non-ssl protocols (https is kind of hardcoded for EOS and netconf and expect uses ssh as transport so...)